mbuild/packing.py

```diff
@@ -122,7 +122,10 @@ def fill_region(compound, n_compounds, region, overlap=0.2, seed=12345):
         compound = [compound]
     if not isinstance(n_compounds, (list, set)):
         n_compounds = [n_compounds]
-    if not any(isinstance(reg, list) for reg in region): # See if region is a single region or list
+    # See if region is a single region or list
+    if type(region) == Box: # Cannot iterate over boxes
```

Comment:
Also `isinstance` is generally the preferred way to check for types in cases like this